core/src/main/java/tc/oc/pgm/map/MapLibraryImpl.java

```diff
@@ -161,8 +161,8 @@ public CompletableFuture<?> loadNewMaps(boolean reset) {
               }
 
               try (Stream<? extends MapSource> stream =
-                  mapSources.stream().flatMap(Function.identity()).parallel().unordered()) {
```

Comment:
This commit https://github.com/PGMDev/PGM/commit/e1ddd1bda55e5884d265b5fd57d96f4e84b6759f broke compilation for me, this fixes that. This does basically the same thing

Reply:
how would this break compilation!?

Reply:
I honestly don't know.
It gave the error: `[164,88] incompatible types: java.util.stream.Stream<java.lang.Object> cannot be converted to java.util.stream.Stream<? extends tc.oc.pgm.api.map.MapSource>`
Which made no sense to me